Driverless AI/machine-learning-interpretability-tutorial/machine-learning-interpretability-tutorial.md

```diff
@@ -41,11 +45,11 @@ In this tutorial, we will generate a machine learning model using an example fin
 
 - Basic knowledge of Machine Learning and Statistics
 
-- Basic knowledge of Driverless AI or doing the [Automatic Machine Learning Introduction with Drivereless AI Test Drive](https://h2oai.github.io/tutorials/automatic-ml-intro-test-drive-tutorial/#0) 
+- Basic knowledge of Driverless AI or doing the [Automatic Machine Learning Introduction with Drivereless AI Test Drive](https://training.h2o.ai/products/tutorial-1a-automatic-machine-learning-introduction-with-driverless-ai) 
 
 - A **Two-Hour Test Drive session** : Test Drive is H2O.ai's Driverless AI on the AWS Cloud. No need to download software. Explore all the features and benefits of the H2O Automatic Learning Platform.
 
-  - Need a **Two-Hour Test Drive** session? Follow the instructions on [this quick tutorial](https://h2oai.github.io/tutorials/getting-started-with-driverless-ai-test-drive/#1) to get a Test Drive session started.  
+  - Need a **Two-Hour Test Drive** session? Follow the instructions on [this quick tutorial](https://training.h2o.ai/products/tutorial-0-getting-started-with-driverless-ai-test-drive) to get a Test Drive session started.  
 
```

Comment:
Please remove the following text from the prerequisites:

Other installations:

Cloud Installation
H2O Driverless AI on AWS Marketplace
H2O Driverless AI on Azure HDInsight
H2O.ai Driverless AI (BYOL) on Google Cloud Platform
Local Installation
Server Installation